learn/video.adoc

```diff
@@ -4,10 +4,21 @@
 :awestruct-priority: 0.8
 :showtitle:
 
-See https://www.youtube.com/playlist?list=PLJY69IMbAdq0uKPnjtWXZ2x7KE1eWg3ns[Geoffrey's OptaPlanner playlist on YouTube]
+See https://www.youtube.com/playlist?list=PLJY69IMbAdq0uKPnjtWXZ2x7KE1eWg3ns[the OptaPlanner playlist on YouTube]
 for a list of all videos.
 
+This is the latest video:
+
 // Using the video_id/playlist syntax (with no video_id).
 // See https://github.com/asciidoctor/asciidoctor/pull/1187.
 // Results in showing the newest video in the playlist.
 video::/PLJY69IMbAdq0uKPnjtWXZ2x7KE1eWg3ns[youtube]
+
+== Make your own video's
```

Comment:
I noticed that you're consistently using the possessive form instead of plural. :-) 
"video's" is short for "video is". That's not what you want to say here.

Reply:
fixed, also fixed demo's